src/model/dds/EndpointModelItem.cpp

```diff
@@ -37,6 +37,8 @@ QVariant EndpointModelItem::data(
             return this->topic();
         case aliveRole:
             return this->alive();
+        case clickedRole:
```

Comment:
NIT: We could use this addition to change this function in a way that it inherits the parent cases. I get in mind two options:
 - call parent `data()` and in case the result is `QVariant()` use the `switch case` for the specific cases of this Subclass (This means to create a void `QVariant` unnecessary, but it is easier)
 - create a new function `data_` in a way that nothing is created in `default` case, and could be inherit from the children classes. So, `data()` calls that function and in case of non finding the attribute, create the void `QVariant`